lib/v2/bioone/journal.js

```diff
@@ -0,0 +1,58 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const journal = ctx.params.journal ?? 'acta-chiropterologica';
+
+    const rootUrl = 'https://bioone.org';
+    const currentUrl = `${rootUrl}/journals/${journal}/current`;
+    const response = await got({
+        method: 'get',
+        url: currentUrl,
+    });
+
+    const $ = cheerio.load(response.data);
+
+    let items = $('.TOCLineItemBoldText')
+        .slice(0, ctx.params.limit ? parseInt(ctx.params.limit) : 10)
```

Comment:
`ctx.query.limit`. 
`ctx.params.limit` has no impact on the items count as it will be always 10 as `ctx.params.limit` is always undefined.

Reply:
Thanks for review 😊